### PR TITLE
Revert "Add LED strip sliders"

### PR DESCRIPTION
**Pull request by haslinghuis**
Reverts betaflight/betaflight-configurator#3531

Firmware PR is not merget yet and involves MSP changes possible corrupting configuration.

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Review by VitroidFPV — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Pull request merged**
